e2e/test/scenarios/filters/reproductions/29094-non-boolean-custom-expressions.cy.spec.js

```diff
@@ -0,0 +1,35 @@
+import {
+  enterCustomColumnDetails,
+  getNotebookStep,
+  popover,
+  restore,
+  startNewQuestion,
+} from "e2e/support/helpers";
+
+describe("issue 29094", function () {
```

Comment:
```suggestion
describe("issue 29094", () => {
```

Please change this to an arrow function. @kamilmielnik

Reply:
It's just a convention we've been following for hundreds of other E2E tests. I mean - we can document it and even enforce the lint rule for it.

Using an arrow function prevents you doing silly things like assigning some value to a variable (`foo`) and then latter trying to access that using `this.foo`.

To sum it up:
There is nothing inherently wrong with using `function` instead of an arrow function, but the latter prevents some less-than-ideal hacks. The rest of the E2E code base is using arrow functions.

Reply:
c7ee4698adb1b77b07776620d982a8149f127289

Reply:
#39512